<launch>
    <node name="detection_right_params_load" pkg="dynamic_reconfigure" type="dynparam" args="load detection $(find camera_localization)/params/config.yaml" />
    <node name="detection" pkg="camera_localization" type="camera_localization_node">
        <remap from="odom/0" to="/communication/gps/0" />
        <remap from="odom/1" to="/communication/gps/1" />
        <remap from="odom/2" to="/communication/gps/2" />
        <remap from="odom/3" to="/communication/gps/3" />
        <remap from="odom/4" to="/communication/gps/4" />
        <remap from="odom/5" to="/communication/gps/5" />
        <remap from="odom/6" to="/communication/gps/6" />
        <remap from="odom/7" to="/communication/gps/7" />
        <remap from="odom/8" to="/communication/gps/8" />
        <remap from="odom/9" to="/communication/gps/9" />
        <remap from="odom/10" to="/communication/gps/10" />
        <remap from="odom/11" to="/communication/gps/11" />
        <remap from="odom/12" to="/communication/gps/12" />
        <remap from="odom/13" to="/communication/gps/13" />
        <remap from="odom/14" to="/communication/gps/14" />
        <remap from="odom/15" to="/communication/gps/15" />
        <remap from="odom/16" to="/communication/gps/16" />
        <remap from="odom/17" to="/communication/gps/17" />
        <remap from="odom/18" to="/communication/gps/18" />
        <remap from="odom/19" to="/communication/gps/19" />
        <remap from="odom/20" to="/communication/gps/20" />
        <remap from="odom/21" to="/communication/gps/21" />
        <remap from="odom/22" to="/communication/gps/22" />
        <remap from="odom/23" to="/communication/gps/23" />
        <remap from="odom/24" to="/communication/gps/24" />
        <remap from="odom/25" to="/communication/gps/25" />
        <remap from="odom/26" to="/communication/gps/26" />
        <remap from="odom/27" to="/communication/gps/27" />
        <remap from="odom/28" to="/communication/gps/28" />
        <remap from="odom/29" to="/communication/gps/29" />
        <remap from="odom/30" to="/communication/gps/30" />
        <remap from="odom/31" to="/communication/gps/31" />
        <remap from="odom/32" to="/communication/gps/32" />
        <remap from="odom/33" to="/communication/gps/33" />
        <remap from="odom/34" to="/communication/gps/34" />
        <remap from="odom/35" to="/communication/gps/35" />
        <remap from="odom/36" to="/communication/gps/36" />
        <remap from="odom/37" to="/communication/gps/37" />
        <remap from="odom/38" to="/communication/gps/38" />
        <remap from="odom/39" to="/communication/gps/39" />
        <remap from="odom/40" to="/communication/gps/40" />
        <remap from="odom/41" to="/communication/gps/41" />
        <remap from="odom/42" to="/communication/gps/42" />
        <remap from="odom/43" to="/communication/gps/43" />
        <remap from="odom/44" to="/communication/gps/44" />
        <remap from="odom/45" to="/communication/gps/45" />
        <remap from="odom/46" to="/communication/gps/46" />
        <remap from="odom/47" to="/communication/gps/47" />
        <remap from="odom/48" to="/communication/gps/48" />
        <remap from="odom/49" to="/communication/gps/49" />
        <remap from="odom/50" to="/communication/gps/50" />

        <!-- car marker translation in respect to base_link -->
        <param name="front_marker_translation_x" value="0.00" />
        <param name="front_marker_translation_y" value="0.00" />
        <param name="front_marker_translation_z" value="0.30" />

        <param name="marker_1_x" value="0.175" />
        <param name="marker_1_y" value="0.135" />
        <param name="marker_1_z" value="0.0" />

        <param name="marker_4_x" value="2.175" />
        <param name="marker_4_y" value="0.135" />
        <param name="marker_4_z" value="0.0" />

        <param name="marker_6_x" value="4.175" />
        <param name="marker_6_y" value="0.135" />
        <param name="marker_6_z" value="0.0" />

        <param name="marker_2_x" value="5.825" />
        <param name="marker_2_y" value="0.135" />
        <param name="marker_2_z" value="0.0" />

        <param name="marker_7_x" value="0.175" />
        <param name="marker_7_y" value="4.175" />
        <param name="marker_7_z" value="0.0" />

        <param name="marker_5_x" value="2.175" />
        <param name="marker_5_y" value="4.175" />
        <param name="marker_5_z" value="0.0" />

        <param name="marker_3_x" value="4.175" />
        <param name="marker_3_y" value="4.175" />
        <param name="marker_3_z" value="0.0" />

        <param name="marker_0_x" value="5.825" />
        <param name="marker_0_y" value="4.175" />
        <param name="marker_0_z" value="0.0" />
    </node>
</launch>
